frontend/src/app/models/ml-model.ts

```diff
@@ -0,0 +1,259 @@
+// Copyright 2023 Google Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { Pipeline } from './pipeline';
+
+export enum Type {
+  LOGISTIC_REG = 'LOGISTIC_REG',
+  BOOSTED_TREE_REGRESSOR = 'BOOSTED_TREE_REGRESSOR'
+}
+
+export enum UniqueId {
+  CLIENT_ID = 'CLIENT_ID',
+  USER_ID = 'USER_ID'
+}
+
+export enum Source {
```

Comment:
These sources are used here and in ml-models.service.ts so can they be defined in another file that can be imported or is it better to leave as is given the context?

Reply:
ml-models.service.ts doesn't have anything except backend endpoint calls. Are you certain you're looking at the most recent code?

Reply:
Understood.